maintainers-guide.md

```diff
@@ -195,34 +195,40 @@ issues.
     * Contact: [gregestren](https://github.com/gregestren)
+*   [`team-Core`](https://github.com/bazelbuild/bazel/labels/team-Core): Issues for Core team
+    * Contact: ?
 *   [`team-EngProd`](https://github.com/bazelbuild/bazel/labels/team-Engprod): Issues for EngProd team (CI, release, distribution)
     * Contact: [philwo](https://github.com/philwo)
 *   [`team-ExternalDeps`](https://github.com/bazelbuild/bazel/labels/team-ExternalDeps): Issues for External Deps team (remote repositories,
     WORKSPACE file)
     * Contact: [laurentlb](https://github.com/laurentlb)
+*   [`team-Front-End`](https://github.com/bazelbuild/bazel/labels/team-Front-End): Issues for the Bazel Front End team
+    * Contact: [laurentlb](https://github.com/laurentlb)
 *   [`team-Local-Exec`](https://github.com/bazelbuild/bazel/labels/team-Local-Exec): Issues for Execution (Local) team
     * Contact: [jmmv](https://github.com/jmmv)
-*   [`team-Remote-Exec`](https://github.com/bazelbuild/bazel/labels/team-Remote-Exec): Issues for Execution (Remote) team
-    * Contact: [buchgr](https://github.com/buchgr)
 *   [`team-Performance`](https://github.com/bazelbuild/bazel/labels/team-Performance): Issues for Bazel Performance team
     * Contact: [meisterT](https://github.com/meisterT)
-*   [`team-Product`](https://github.com/bazelbuild/bazel/labels/team-Product): Issues for the Product team
-    * Contact: [dslomov](https://github.com/dslomov)
+*   [`team-Remote-Exec`](https://github.com/bazelbuild/bazel/labels/team-Remote-Exec): Issues for Execution (Remote) team
+    * Contact: ?
 *   [`team-Rules-CPP`](https://github.com/bazelbuild/bazel/labels/team-Rules-CPP): Issues for C++ rules
-    * Contact: [hlopko](https://github.com/hlopko)
+    * Contact: [lberki](https://github.com/lberki)
 *   [`team-Rules-Java`](https://github.com/bazelbuild/bazel/labels/team-Rules-Java): Issues for Java rules
-    * Contact: [hlopko](https://github.com/hlopko)
+    * Contact: [lberki](https://github.com/lberki)
 *   [`team-Rules-Python`](https://github.com/bazelbuild/bazel/labels/team-Rules-Python): Issues for the native Python rules
-    * Contact: [brandjon](https://github.com/brandjon)
+    * Contact: [lberki](https://github.com/lberki)
 *   [`team-Rules-Server`](https://github.com/bazelbuild/bazel/labels/team-Rules-Server): Issues for serverside rules included with Bazel
-    * Contact: [hlopko](https://github.com/hlopko)
+    * Contact: [lberki](https://github.com/lberki)
 *   [`team-Starlark`](https://github.com/bazelbuild/bazel/labels/team-Starlark): Issues for Starlark language + Build API
     * Contact: [laurentlb](https://github.com/laurentlb)
 *   [`team-Windows`](https://github.com/bazelbuild/bazel/labels/team-Windows): Issues for Windows team
-    * Contact: [laszlocsomor](https://github.com/laszlocsomor)
+    * Contact: [philwo](https://github.com/philwo)
```

Comment:
@meteorcloudy ?

Reply:
Yeah, you can put me there. Philipp already has many things to own.

Reply:
Done. Thanks.